docs/README.md

```diff
@@ -1,3 +1,9 @@
 # Sidecar OpenApi UI
 
-Proposed OpenApi spec for [substrate-api-sidecar](https://github.com/paritytech/substrate-api-sidecar) v1.
+OpenApi spec for [substrate-api-sidecar](https://github.com/paritytech/substrate-api-sidecar).
+
+## Contributing:
+
+After updating the docs, ensure to run `yarn build:docs` from the root directory of this repository.
```

Comment:
That is actually running
> "build:docs": "(cd docs && yarn && yarn build)",
Why not to add  here:  
or `yarn && yarn build` from docs folder

Reply:
I think it's good to give users a single way of doing this. Leaves us free to move&change the docs building step at any time.